web/docs/guides/database/extensions/pg-cron.mdx

```diff
@@ -0,0 +1,109 @@
+---
+id: pgcron
+title: "pg_cron: Postgres Schedular"
```

Comment:
spelling error in scheduler

Reply:
(needs to be fixed everywhere)

Reply:
@kiwicopple All grammar errors have been fixed